plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2135,7 +2135,7 @@ fc::variant read_only::get_block_header_state(const get_block_header_state_param
 
 void read_write::push_block(read_write::push_block_params&& params, next_function<read_write::push_block_results> next) {
    try {
-      app().get_method<incoming::methods::block_sync>()(std::make_shared<signed_block>(std::move(params)), {});
+      app().get_method<incoming::methods::block_sync>()(std::make_shared<signed_block>(params, true), {});
```

Comment:
After the addition of `signed::signed_block( signed_block_v0&&, bool legacy )` which we already need for other reasons (https://github.com/EOSIO/eos/pull/8891#discussion_r400342768), this should use `std::move(params)`.